### PR TITLE
new model - data from storage

### PR DESCRIPTION
**Pull request by mkeskells**
run the data from a DB ( csvs) containing the info that we need for the analysis

### CODE REVIEW

File: rules/src/main/scala/scalaclean/model/impl/v2/Parser.scala [hunk 1]
```diff
@@ -18,7 +18,13 @@ class ParserImpl extends ParseModel {
     writer.close()
   }
 
-  override def asProjectModel: ProjectModel = ???
+  override def asProjectModel: ProjectModel = {
+    val projectsRoot = Paths.get("s:/temp")
```
Comment (mkeskells) [inline rules/src/main/scala/scalaclean/model/impl/v2/Parser.scala]: some hard coded stuff until integrated with a build tool

File: rules/src/main/scala/scalafix/scalaclean/FixUtils.scala [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import scalafix.internal.v1.InternalSemanticDoc
+import scalafix.v1.SemanticDocument.Error
+import scalafix.v1.{SemanticDocument, SyntacticDocument}
+
+import scala.meta.inputs.Input
+import scala.meta.internal.semanticdb.TextDocuments
+import scala.meta.internal.symtab.{GlobalSymbolTable, SymbolTable}
+import scala.meta.io.{AbsolutePath, Classpath, RelativePath}
+
+object FixUtils {
```
Comment (mkeskells) [inline rules/src/main/scala/scalafix/scalaclean/FixUtils.scala]: workarounds for the scalafix API where things are private

File: rules/src/main/scala/scalafix/scalaclean/FixUtils.scala [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  def loadSemanticDoc(project: Project, path: Path) : SemanticDocument = {
+    val syntacticDoc = loadSyntaticDocument(project, path)
+    val rel = if (path.isAbsolute) project.absolutePath.relativize(path) else path
+    println(rel)
+    println(project.relativePath)
+    println(project.relativePath.resolve(rel))
+    fromPath(syntacticDoc, RelativePath(project.relativePath.resolve(rel)), project.classloader, project.symbolTable)
+  }
+
+  private[scalafix] def fromPath(
```
Comment (mkeskells) [inline rules/src/main/scala/scalafix/scalaclean/FixUtils.scala]: SemanticDocument.fromPath doesnt work on windows \ vs / in the path

**Comment by mkeskells** [comment]
cc: @hrhino

**Comment by hrhino** [comment]
On the train. I'll have a look once I'm home.

**Pull request merged**